### PR TITLE
cmus: update command

### PR DESCRIPTION
**Pull request by syrofoam**
Mutch needed kb shortcuts.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/cmus.md [hunk 1]
```diff
@@ -11,6 +12,14 @@
```
Comment (zdroid) [inline pages/linux/cmus.md]: Your addition itself is good, but as we're here, I suggest to change `file/directory` above to `file or directory`.

File: pages/linux/cmus.md [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 `s`
 
-- Turn volume up/down and seeking
+- Turn volume up/down and seeking:
```
Comment (zdroid) [inline pages/linux/cmus.md]: I can't understand the meaning behind this, and it also seems `seeking` doesn't agree gramatically with the rest of the sentence. What's the context of `seeking` here?

File: pages/linux/cmus.md [hunk 3]
```diff
@@ -2,14 +2,19 @@
 
 > Commandline Music Player.
```
Comment (sbrl) [inline pages/linux/cmus.md]: Perhaps this will read better?
```suggestion
> Command-line music player.
```

File: pages/linux/cmus.md (hunk 2 above)
Comment (agnivade) [reply pages/linux/cmus.md]: The up/down arrow keys are for volume. And the left/right keys are for seeking.

I would separate the two into separate examples.

File: pages/linux/cmus.md [hunk 4]
```diff
@@ -2,14 +2,19 @@
...
 > Use arrow keys to navigate, `<enter/return>` to select, and numbers 1-8 switch between different views.
+> More information: <https://cmus.github.io/>.
 
-- Open cmus from specified directory:
+- Add file/directory to library:
```
Comment (agnivade) [inline pages/linux/cmus.md]: The first example should always invoke the command. Otherwise, one wouldn't know where to enter the commands.

**Pull request closed without merging**